packages/prism-theme-odyssey/src/utils/CustomPropertyInspector.js

```diff
@@ -0,0 +1,122 @@
+        properties[name] = value
+      }
+
+      if (Storage.getItem(name)) {
+        document.documentElement.style.setProperty(name, value);
+      }
+    }
+    return properties
+  }
+
+
+  renderInspector() {
+    const { properties } = this
+    const inspector = document.createElement('form');
+    inspector.classList.add('custom-properties-inspector');
+    inspector.style = `
+      position: sticky;
+      top: 0;
+      border-right: 5px solid #f5f5f6;
+      padding:  3.42857rem 1.71429rem;
+      max-height: 100vh;
+      overflow: scroll;
+      min-width: 16rem;
+    `
+
+    const inspectorTitle = document.createElement('h5');
+    inspectorTitle.innerHTML = "Custom Properties Inspector";
+    inspectorTitle.style="margin-bottom: 2rem;"
+    inspector.appendChild(inspectorTitle)
+
+    const inspectorReset = document.createElement('button');
+    inspectorReset.classList.add('ods-button', 'is-ods-button-danger', 'is-ods-button-full-width');
+    inspectorReset.innerHTML = "Reset Theme";
+
+    document.body.style = "display: flex;"
+
+    for (const [key, value] of Object.entries(properties)) {
+      const fieldset = document.createElement('fieldset');
+      const fieldsetFlex = document.createElement('div');
+      const input = document.createElement('input');
+      const label = document.createElement('label');
+
+      // Set Odyssey Fieldset
+      fieldset.classList.add('ods-fieldset')
+
+      // Set Odyssey Fieldset Flex
+      fieldsetFlex.classList.add('ods-fieldset-flex')
+
+      // Set Odyssey Label
+      label.classList.add('ods-label');
+      label.innerHTML = key;
+      
+      // Set Odyssey Text Input
+      input.setAttribute('type', 'text');
+      input.classList.add('ods-text-input');
+      input.name = key
+      input.value = value.replace(/([A-Z]+)/g, " $1").replace(/([A-Z][a-z])/g, " $1");
+
+      // Append all the elements
+      fieldset.appendChild(fieldsetFlex)
+      fieldsetFlex.appendChild(label);
+      fieldsetFlex.appendChild(input);
+      inspector.appendChild(fieldset);
+      inspector.appendChild(inspectorReset)
+    }
+
+    if (Object.keys(properties).length === 0 && properties.constructor === Object) {
+      const errorNotice = document.createElement('div');
+      errorNotice.innerHTML = `
+        <div>
+          <div style="font-size: 6rem; text-align: center;" aria-label="No good">⚠️</div>
+          <p>The Custom Property Inspector has nothing to show. This is probably because you are <a href="https://github.com/w3c/csswg-drafts/issues/1316">using Google Chrome.</a></p>
+          <p>Try this page on Safari or Firefox!</p>
+        </div>
+      `
+      inspector.appendChild(errorNotice)
+    }
+
+    // Prepend the inspector
+    document.body.prepend(inspector);
+
+    inspectorReset.addEventListener('click', () => {
+      Storage.clear();
+      this.setCustomProperties();
+    })
+
+    inspector.addEventListener('keyup', (e)=> {
```

Comment:
keyup on the entire form? This works reliably to target only inputs and doesn't capture other event noise for tab navigation or arrow key cursor movement? I would think we'd want to do some introspection here to determine if the event came from one of the inputs that controls a meaningful style yea?

Reply:
In the current (very specific) use case, the only "tabbable" (interactive) elements within the inspector are the input elements and reset button. If we want to make this truly portable for different usecases (I can think of several as we really dig in to theming) this can and should be re-evaluated.

For now, I have added a comment If you think this is worth fixing prior to merge let me know. I'm happy to give it a closer look!